setup.py

```diff
@@ -44,5 +44,5 @@
       zip_safe=False,
       install_requires=['pyyaml'],
       tests_require=['pytest'],
-      python_requires='>=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*',
+      python_requires='>=3.4,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*',
```

Comment:
You can remove everything after the `3.4` here since they are included in the new condition. And actually can you make this `3.6`?

Reply:
Done